### PR TITLE
people section on homepage now significantly smaller

### PR DESCRIPTION
**Pull request by wrightmartin**
Now looks like this:

screen shot 2016-08-09 at 17 25 17

### CODE REVIEW

**Comment by BenJam** [comment]
👍

File: ainow/templates/ainow/index.html [hunk 1]
```diff
@@ -160,6 +160,7 @@
             <p class="person__job-title">Co-founder, University of Washington Tech Policy Lab, Assistant Professor, University of Washington School of Law</p>
         </div>
      </div>
+     <a href="schedule/workshop/attendees" class="btn btn-default homepage-people__full-link">All attendees and speakers</a>
```
Comment (wrightmartin) [inline ainow/templates/ainow/index.html]: @stevenday is there a correct way to build this link?

**Comment by BenJam** [comment]
👍 👍

File: ainow/templates/ainow/index.html (hunk 1 above)
Comment (stevenday) [inline ainow/templates/ainow/index.html]: Yep - `{% url 'attendees' 'workshop' %}` should do the trick

Comment (stevenday) [inline ainow/templates/ainow/index.html]: I've done that in  https://github.com/mysociety/ainow/pull/48/commits/627e87258469fd06b2a4066325716dba9d698c3b

**Pull request merged**